### PR TITLE
fixed the get error message

### PR DESCRIPTION
**Pull request by msenmurugan**
#810

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/823#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/cmd_get.go [hunk 1]
```diff
@@ -177,7 +177,7 @@ var getBlackDuckRootKeyCmd = &cobra.Command{
 		if err != nil {
 			return fmt.Errorf("error writing to file '%s' due to %+v", fileName, err)
 		}
-		log.Infof("successfully created the master key in file '%s' for Black Duck '%s' in namespace '%s'", fileName, blackDuckName, blackDuckNamespace)
+		log.Infof("successfully received the master key and generate the file '%s' for Black Duck '%s' in namespace '%s'", fileName, blackDuckName, blackDuckNamespace)
```
Comment (rrati) [inline pkg/synopsysctl/cmd_get.go]: successfully retrieved the master key and wrote it to file '%s' for ....

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/823#" title="Author self-approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/823#pullrequestreview-285662388" title="Approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/cmd_get.go (hunk 1 above)
Comment (msenmurugan) [reply pkg/synopsysctl/cmd_get.go]: done

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/823#" title="Author self-approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/823#pullrequestreview-285667408" title="LGTM">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**